package-scripts.js

```diff
@@ -54,13 +62,21 @@ module.exports = {
          *   npm start "test.automated --bail ./test/dao.js"
          *
          */
-        default: "mocha --require babel-register --require babel-polyfill --require chai --timeout 999999",
-        bail: 'nps "test.automated --bail"'
+        default: series(
+          "nps test.build",
+          "mocha --require babel-register --require babel-polyfill --require chai --timeout 999999"),
+        bail: series(
+          'nps "test.automated --bail"'
+        ),
       },
-      watch: 'nps "test.automated --watch"',
-      bail: 'nps test.automated.bail',
+      watch: series(
+        'nps "test.automated --watch"'
+      ),
+      bail: (
+        'nps test.automated.bail'
+      ),
       ganache: {
-        run: `ganache-cli -l ${config.get("gasLimit")} --account="0x0191ecbd1b150b8a3c27c27010ba51b45521689611e669109e034fd66ae69621,9999999999999999999999999999999999999999999" --account="0x00f360233e89c65970a41d4a85990ec6669526b2230e867c352130151453180d,9999999999999999999999999999999999999999999" --account="0x987a26abca7432016104ce2f24ce639340e25afe06ac69f68791399e7a5d1028,9999999999999999999999999999999999999999999" --account="0x89af34b1b7347834048b99423dad174a14bf14540d720d72c16ae92e94b987cb,9999999999999999999999999999999999999999999" --account="0xc867be647eb2bc51e4c0d61066859875cf3634fe949b6f5f85c69ab90e485b37,9999999999999999999999999999999999999999999" --account="0xefabcc2377dee5e51b5a9e65a3854aec85fbbec3cb584d8ad4f9679869fb33c6,9999999999999999999999999999999999999999999"`,
+        run: `ganache-cli -l ${Config.gasLimit} --account="0x0191ecbd1b150b8a3c27c27010ba51b45521689611e669109e034fd66ae69621,9999999999999999999999999999999999999999999" --account="0x00f360233e89c65970a41d4a85990ec6669526b2230e867c352130151453180d,9999999999999999999999999999999999999999999" --account="0x987a26abca7432016104ce2f24ce639340e25afe06ac69f68791399e7a5d1028,9999999999999999999999999999999999999999999" --account="0x89af34b1b7347834048b99423dad174a14bf14540d720d72c16ae92e94b987cb,9999999999999999999999999999999999999999999" --account="0xc867be647eb2bc51e4c0d61066859875cf3634fe949b6f5f85c69ab90e485b37,9999999999999999999999999999999999999999999" --account="0xefabcc2377dee5e51b5a9e65a3854aec85fbbec3cb584d8ad4f9679869fb33c6,9999999999999999999999999999999999999999999"`,
```

Comment:
do we really need the accounts here ?

Reply:
I don't know.  I know this ensures that the accounts have plenty of gas.  If this is an issue we should create a separate issue for it.

Reply:
it gives them all a ton of ETH, maybe there is a different way to do  this. its also nice to have consistent accounts for dev and testing purposes